editions/tw5.com/tiddlers/webserver/WebServer API_ Get All Tiddlers.tid

```diff
@@ -15,8 +15,19 @@ Parameters:
 * ''filter'' - filter identifying tiddlers to be returned (optional, defaults to "[all[tiddlers]!is[system]sort[title]]")
 * ''exclude'' - comma delimited list of fields to excluded from the returned tiddlers (optional, defaults to "text")
 
-Response:
+An optional list of permitted filters may be configured in [[tiddlywiki.info|tiddlywiki.info Files]] in the config setting `server-get-tiddlers-allowed-filters`. If the setting is missing then all filters are permitted. For example, the following setting explicitly permits the filter used by the TiddlyWeb adaptor plugin:
```

Comment:
I think security should be default. Extended rights should be optional.
just a thought

Reply:
Hi @pmario that's what I've been thinking overnight too. I'll address it shortly.